src/api/api.cpp

```diff
@@ -442,7 +451,6 @@ struct migraphx_optimals
     std::set<size_t> object;
 };
 
-extern "C" struct migraphx_dynamic_dimension;
```

Comment:
These changes will get overwritten when doing `make generate`.

Reply:
How can I make them permanent?

Reply:
You need to modify tool/api.py that is the script generating this.